src/govuk/components/hide-this-page/README.md

```diff
@@ -0,0 +1,15 @@
+# Hide this page
```

Comment:
Sorry to be picky as I know this isn't ready for production yet, but I've noticed in the past that it's really easy to forget to update this file to talk about the correct component (it's normally copied + pasted) and so sometimes we notice it a way down the line. So just an early heads up that this file still references the inset text component.